### PR TITLE
Update release notes

### PR DESCRIPTION
**Pull request by ankoshe1**
Updated release notes.

### CODE REVIEW

**Comment by PRMerger16** [comment]
@ankoshe1 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: azure-stack/partner/azure-stack-vaas-release-notes.md [hunk 1]
```diff
@@ -24,6 +24,28 @@ ms.lastreviewed: 10/28/2019
 
 This article has the release notes for Azure Stack Validation as a Service.
 
+## Version 5.1.49
```
Comment (gadeweever) [inline azure-stack/partner/azure-stack-vaas-release-notes.md]: Service release versions are added to the documentation. The current build of the service is 4.3.5.3.

**Comment by PRMerger9** [comment]
@ankoshe1 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by mattbriggs** [comment]
#sign-off

**Comment by PRMerger10** [comment]
@ktoliver : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**